plugins/docker/docker_

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+#
+# Plugin to graph docker container and storage.
+#
+# GNU GPL Are Tysland & Philip Gabrielsen
+
+TYPE=$( echo $0 | cut -f2 -d'_' )
+
+case $1 in
+autoconf)
```

Comment:
You need to announce the `autoconf` (and `suggest`) capability in the documentation header.